packages/subgraph/src/mappings/superToken/ropstenSuperToken.ts

```diff
@@ -0,0 +1,55 @@
+import { Address } from "@graphprotocol/graph-ts";
```

Comment:
this file should be deleted.

Also for generated file, it would be good to add a comment in the beginning showing that it is a generated file, do not edit manually.

Reply:
oops, I forgot to remove this, we just have one generated file-`addresses.ts` (which is not committed per .gitignore) which hardcodes the addresses for the different networks. it adds a few extra lines to the the package.json (9), but removes a bunch of files. thanks to @kasparkallas for the suggestion.